pkg/config/config.go

```diff
@@ -380,6 +389,23 @@ func NewConfig(userConfigPath string) (*Config, error) {
 		return nil, err
 	}
 
+	// read the configs from libpod.conf if exists
+	if hasLibpodConf() {
+		configFromLibpod, err := newLibpodConfig(config)
+		if err != nil && !os.IsNotExist(err) {
+			logrus.Errorf("error reading libpod.conf: %v", err)
+		}
+		if err == nil {
+			config.libpodToContainersConfig(configFromLibpod)
+			config.checkCgroupsAndAdjustConfig()
+			config.addCAPPrefix()
+
+			if err := config.Validate(); err == nil {
```

Comment:
This break should not be here.  The code should continue to read the containers.conf files on the system. 

We want to have the following merged in this order
containers/common defaults
libpod.conf merged
/usr/share/containers/containers.conf
/etc/containers/containers.conf
~/.config/containers/containers.conf
All merged together if they exists.

libpod.conf does not short cut the merge.